R/init.R

```diff
@@ -1,7 +1,8 @@
 if (interactive() &&
   Sys.getenv("RSTUDIO") == "" &&
   Sys.getenv("TERM_PROGRAM") == "vscode") {
-  if (requireNamespace("jsonlite", quietly = TRUE)) local({
+  if (requireNamespace("jsonlite", quietly = TRUE) &&
+      requireNamespace("rlang", quietly = TRUE)) local({
```

Comment:
```suggestion
  is_pkg_installed <- function(x) {
    not_installed <- x[!vapply(x,  requireNamespace, quietly = TRUE, FUN.VALUE = logical(1L))]
    if (length(not_installed)) message("VSCode-R depends on R package: ", toString(not_installed), ". Please install manually in order to use VSCode-R.")
    length(not_installed) == 0L
  }
  if (is_pkg_installed(c("jsonlite", "rlang"))) local({
```

Reply:
These functions will leak to user global environment. Let me put them in a local environment to avoid that.